### PR TITLE
add the ability to fetch repositories for a specific installation and user

### PR DESCRIPTION
**Pull request by bdelbasso**
https://developer.github.com/early-access/integrations/user-identification-authorization/#check-which-installations-resources-a-user-can-access

@Nyholm @acrobat

### CODE REVIEW

File: lib/Github/Api/CurrentUser.php [hunk 1]
```diff
@@ -178,4 +178,15 @@ public function installations(array $params = array())
...
+
+    /**
+     * @link https://developer.github.com/v3/integrations/installations/#list-repositories-accessible-to-the-user-for-an-installation
+     *
+     * @param string $installationId  the ID of the Installation
+     * @param array $params
+     */
+    public function repositoriesByInstallation($installationId, array $params = array())
+    {
+        return $this->get("/user/installations/$installationId/repositories", array_merge(array('page' => 1), $params));
```
Comment (m1guelpf) [inline lib/Github/Api/CurrentUser.php]: Why is page hardcoded?

Comment (acrobat) [inline lib/Github/Api/CurrentUser.php]: Page is only hardcoded when you don't pass a page array key in the params array. So this is the default value

**Comment by bdelbasso** [comment]
@acrobat @Nyholm does one of you has the ability to trigger the style check again? Looks like it's stuck.

**Comment by acrobat** [comment]
It looks like the styleci job for this PR was not started. You could rebase and force push which should trigger a new build

File: lib/Github/Api/CurrentUser.php (hunk 1 above)
Comment (Nyholm) [inline lib/Github/Api/CurrentUser.php]: Lets use sprintf here. Also, do single quotes.

Comment (bdelbasso) [inline lib/Github/Api/CurrentUser.php]: Will do.

**Comment by bdelbasso** [comment]
@Nyholm I'm not sure about your last comment (the Accept header). I think it's outside of the scope of this PR.

If I want to call any route regarding integrations (any method in `Integrations.php` - which are not discussed here), I [also need](https://developer.github.com/v3/integrations/) to set the Accept header. Which is possible in the constructor of the Client (second parameter `$apiVersion`).

I see two benefits of not adding it here:
- the developer is aware that this route is _early access_: if the appropriate api version is not set, then he'll get an error from Github API.
- when the route is released publicly, we don't have to change anything.

**Comment by Nyholm** [comment]
> If I want to call any route regarding integrations (any method in Integrations.php - which are not discussed here), I also need to set the Accept header. Which is possible in the constructor of the Client (second parameter $apiVersion).

I did not know. Thanks. 

> I see two benefits of not adding it here

I do agree with your reasons, but the configure method is needed for consistency. 

> I think it's outside of the scope of this PR. 

Yeah, since it is needed for the existing endpoints as well. I created a new issue for that https://github.com/KnpLabs/php-github-api/issues/580

**Comment by Nyholm** [comment]
Thank you for this PR.

**Pull request merged**